.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+AllCops:
+  TargetRubyVersion: 2.3
+
+Style/FrozenStringLiteralComment:
```

Comment:
Reckon you can drop this - looks like it's included in https://github.com/alphagov/govuk-lint/blob/master/configs/rubocop/other-style.yml by default.